linode/design-language-system
# feat: Update Structure for Figma Variables

## Description 📝
We needed to update the structure to have the tokens studio plugin create Figma variables more reliably. We grouped the themes and added better names. In addition we needed to update the JSON structure slightly.

## Major Changes 🔄
- Bumped package to `v1.2.0`
    - Our CI needed some fixing since it ran our publish command before this was merged
- Added Storybook
    - Global Color Swatches 
- Removed plural `Colors` and `Actions`
- Global, Alias and Component tokens have additional hierarchies for their JSON
- Updated the Themes & Metadata
   - Dark themes inherit from light, so we only need to provide overrides
- Added components
   - Appbar
   - Container
   - Table

## Testing
1. `yarn generate`
2. Copy `dist` into project `node_modules`
3. Observe changes

## Running Storybook
1. `yarn storybook`

![Screenshot 2023-12-29 at 12 54 53 AM](https://github.com/linode/design-language-system/assets/125309814/46fe252d-86b5-4489-bf5c-8ac4db449aa8)
